
---
uint8[] faces
uint8[] picture
string position
string result
string user_find
